internal/services/controller/handlers/filters/onlypodoom/filter.go

```diff
@@ -0,0 +1,17 @@
+package onlypodoom
```

Comment:
Sounds a bit complex for package name. Maybe 'oomevents' would fit?

Reply:
yep, updated

Reply:
renamed